README.md

```diff
@@ -24,15 +24,15 @@ from which data can be extracted. The rippled node does not need to be running o
 2. Access to one or more rippled nodes. Can be local or remote.
 
 ## Building
-clio is built with cmake. clio requires c++20, and boost 1.75.0 or later. protobuf v2 is required and must be manually built on systems that ship with v3, including Debian 11 and Ubuntu 21.10.
+clio is built with cmake. clio requires c++20, and boost 1.75.0 or later. protobuf v2 or v3 can both be used, however, users with older systems will need to update to GCC 8 or later and cmake 3.16.3 or later.
 Use these instructions to build a clio executable from source. These instructions were tested on Ubuntu 20.04 LTS.
 
 ```
 1. sudo apt-get update
 2. sudo apt-get -y upgrade
```

Comment:
Why upgrade?

Reply:
There's no reason these instructions _require_ an upgrade so let the user upgrade their system. If it ain't broke, don't fix it?

If any version of protobuf can be used and if you don't have it installed it gets built locally, why even mention it?
I know there was a few weird issues identifying protobuf deps but that's fixed.

I'm for succinctness here obviously ;)

Reply:
I personally find it a tiny bit easier to have some version info in the README, even though it isn't strictly necessary. It makes it easier when I'm trying to build on different systems. I agree that protobuf doesn't need to be mentioned explicitly, aside from it being helpful for the few people who are used to the inaccurate info that was in the rippled README.

Reply:
I've removed references to protobuf.